index.js

```diff
@@ -10,16 +10,18 @@ const formatSassVariables = require('./lib/formatSassVariables')
 
 const stylefmt = postcss.plugin('stylefmt', function (options) {
   var paramer = params(options)
-  return function (root) {
-    return paramer(root).then(function (params) {
-      formatComments(root, params)
-      formatAtRules(root, params)
-      formatOrder(root, params)
-      formatRules(root, params)
-      formatSassVariables(root, params)
-      return root
+  return function (root, result) {
```

Comment:
It is ues `result` for result.stylelint.ignored
https://github.com/morishitter/stylefmt/pull/231/files#diff-f5008a4d411fbb1dbee72fa9d191a862R98